CONTRIBUTING.md

```diff
@@ -1,22 +1,46 @@
-# Contributing
-
-Thank you for considering making contributions to the Bera Agent Kit!
-
-Contributing to this repo can mean many things, such as participating in
-discussion or proposing code changes. To ensure a smooth workflow for all
-contributors, the general procedure for contributing has been established:
-
-1. Start by browsing [new issues](https://github.com/Webera-Finance/bera-agent-kit/issues). If you are looking for something interesting or if you have something in your mind, there is a chance it had been discussed.
-   * Looking for a good place to start contributing? How about checking out some [good first issues](https://github.com/Webera-Finance/bera-agent-kit/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or [bugs](https://github.com/Webera-Finance/bera-agent-kit/issues?q=is%3Aopen+is%3Aissue+label%3Abug)?
-2. Determine whether a GitHub issue or discussion is more appropriate for your needs:
-   1. If the issue you want addressed is a specific proposal or a bug, then open a [new issues](https://github.com/Webera-Finance/bera-agent-kit/issues).
-   2. Review existing [issues](https://github.com/Webera-Finance/bera-agent-kit/issues) to find an issue you'd like to help with.
-3. Participate in thoughtful discussion on that issue.
-4. If you would like to contribute:
-   1. Ensure that the proposal has been accepted.
-   2. Ensure that nobody else has already begun working on this issue. If they have,
-      make sure to contact them to collaborate.
-   3. If nobody has been assigned to the issue and you would like to work on it,
-      make a comment on the issue to inform the community of your intentions
-      to begin work.
-5. To submit your work as a contribution to the repository follow standard GitHub best practices. Please open a pull request!
\ No newline at end of file
+# Contributing to Bera Agent Kit
+
+Thank you for considering contributing to the Bera Agent Kit! Your contributions help make the project better for everyone.
+
+## Table of Contents
+
+1. [Code of Conduct](CODE_OF_CONDUCT.md)
+2. [Getting Started](GETTING_STARTED.md)
+3. [Reporting Bugs](REPORTING_BUGS.md)
+4. [Suggesting Enhancements](SUGGESTING_ENHANCEMENTS.md)
+5. [Making Changes](MAKING_CHANGES.md)
+6. [Pull Request Process](PULL_REQUEST_PROCESS.md)
+7. [License](LICENSE)
+8. [Вклад на русском](CONTRIBUTING_ru.md)
+
+## Code of Conduct
+
+Please read our [Code of Conduct](CODE_OF_CONDUCT.md) to understand our expectations for participation in the project.
+
+## Getting Started
+
+Follow the steps in [Getting Started](GETTING_STARTED.md) to set up your development environment and start working on the project.
+
+## Reporting Bugs
+
+If you find a bug, please follow the guidelines in [Reporting Bugs](REPORTING_BUGS.md).
+
+## Suggesting Enhancements
+
+Have an idea to improve the project? Please read [Suggesting Enhancements](SUGGESTING_ENHANCEMENTS.md) on how to propose changes.

```

Comment:
Use this link for feature request issue: https://github.com/Webera-Finance/bera-agent-kit/issues/new?assignees=&labels=features&projects=&template=feature-request.yml&title=%5BFeature%5D%3A+